static/src/javascripts/projects/common/modules/identity/sign-in-gate/index.js

```diff
@@ -37,7 +38,7 @@ const canShow: () => Promise<boolean> = () =>
 const show: () => Promise<boolean> = () =>
     new Promise(resolve => {
         // get the test the user is in
-        const test = getTestforMultiTest(tests);
+        const test: ABTest = getTestforMultiTest(tests); // todo what is test is undefined?????
```

Comment:
if test is undefined, maybe it should `return resolve(false)` early, so the game doesn't show, but if the code has reached this point (`canShow` is `true`) then the test must exist